src/components/widgets/lava-front-height-widget.tsx

```diff
@@ -36,7 +28,11 @@ export function LavaFrontHeightWidget({ lavaFrontHeight }: LavaFrontHeightWidget
   return (
     <ValueContainer backgroundColor={kWidgetPanelInfo.right.backgroundColor}>
       <RelativeIconContainer>
-        <img src={lavaFrontIcon} alt={`Lava front height: ${lavaFrontHeight}m`} />
+        <img
+          src={lavaFrontIcon}
+          alt={`Lava front height: ${lavaFrontHeight}m`}
+          style={{ height: "49px", width: "90px" }}
```

Comment:
In JS this can be simplified to:
```suggestion
          style={{ height: 49, width: 90 }}
```